package.json

```diff
@@ -132,6 +132,7 @@
         "alt-container": "^1.0.0",
         "alt-react": "git+https://github.com/bitshares/react.git",
         "bignumber.js": "^4.0.0",
+        "bitshares-ui-style-guide": "git+ssh://git@github.com/bitshares/bitshares-ui-style-guide.git",
```

Comment:
Using ssh here can cause issues for users without ssh access to that repo, like the build servers. Please change it to use regular https instead